packages/ChangesReloaded-Core.package/CRBrowser.class/instance/retreiveChangeRecordsFromChangesFile.st

```diff
@@ -0,0 +1,13 @@
+initializing
+retreiveChangeRecordsFromChangesFile
+
+	| changesFile changesFileCopy start end |
+	changesFile := SourceFiles at: 2.
+	changesFileCopy := changesFile readOnlyCopy.
+	changesFileCopy setConverterForCode.
+	end := changesFileCopy size.
+	start := 0 max: (end - self class numberOfBytesToLoadChangesFrom).
+	
+	^ ChangeList new
```

Comment:
To be discussed in person: I am not sure if that return statement is readable enough. You might think it returns a whole ChangeList, which is not true because of the cascade

Reply:
Oh, also: why do you use `ChangeList` here although we have our `CRChangeList` now?

Reply:
The problem with the CRChangeList is… well it doesn't really do anything yet I believe. Actually, I'm not even sure we're using it.
For now, only the ChangeList contains the logic for loading the ChangeRecords from the changes file.
We're working on changing that in #76 by extracting the logic for loading the changes into a separate ChangeRecordsLoader class that both we and the ChangeList could use.